some more improvments on the gui

JoHnTsIm opened the pull request:
improvments
-------------
1. Now if the user tries to press convert without having select at least one file for convert, its gonna get this pop up window with this message.

![3](https://user-images.githubusercontent.com/84143748/118415240-f9b83e00-b6b1-11eb-97ed-c64a6df665ad.PNG)


2. Now if the user tries to press convert without having seleet a folder to save the converted files, its gonna get this pop up window with this message

![2](https://user-images.githubusercontent.com/84143748/118415309-4734ab00-b6b2-11eb-9f7c-eb088f6ec5a9.PNG)


3. Now if the user tries to press convert without having select a folder to save the converted files and without having select at least one file for convert, its gonna get this window with this message

![1](https://user-images.githubusercontent.com/84143748/118415360-882cbf80-b6b2-11eb-8302-692e79772811.PNG)


4. Now if user do it all and then press convert, its gonna get this pop up window.

![4](https://user-images.githubusercontent.com/84143748/118415400-c88c3d80-b6b2-11eb-85c8-42956c079df3.PNG) 
if the user selects continue the convert its gonna start, otherwise if the user selects cancel the convert its gonna canceled.


Τhe improved code:
```ruby
from pdf2docx import *
from tkinter import *
from tkinter.filedialog import *
from tkinter import filedialog
import sys
import webbrowser

root = Tk()
root.title('PDF_2_Docx Converter')
root.geometry('500x700')
root.config(bg='grey')
root.iconbitmap('icon.ico')


# Opens file explorer
# and let you select choose the pdf file that you want to convert
def pdf_file_location():
    global filenames
    Tk().withdraw()
    filenames = askopenfilenames()
    file_path_pdf_entry.insert(0, filenames)


# Opens file explorer
# and let you choose the folder, that all the converted file or files going to saved.
def docx_folder_location():
    global folder_selected
    after_convert_folder = filedialog.askdirectory()
    Tk().withdraw()
    folder_selected = after_convert_folder
    file_path_docx_entry.insert(0, folder_selected)


# Starts the convert of the file or files
def convert_button_function():

    # function for pop window 1
    def close():
        root2.destroy()

    # function for pop window 2
    def close_2():
        root3.destroy()

    # function for pop window 3
    def close_3():
        root4.destroy()

    # function for pop window 4
    def close_4():
        root5.destroy()

    # function for pop window 5
    def close_5():
        root5.destroy()
        i = 0
        for file in filenames:
            i += 1
            cv = Converter(file)
            cv.convert(folder_selected + '/' + 'Docx' + str(i) + '.docx', start=0, end=None)
            cv.close()

    # Window pop ups #
    if len(file_path_docx_entry.get()) == 0 and len(file_path_pdf_entry.get()) == 0:
        root4 = Tk()
        root4.title('Neither files or folder selected')
        root4.geometry('400x100')
        root4.config(bg='grey')
        root4.iconbitmap('icon.ico')

        """Labels"""
        selected_folder = Label(root4, text='Select PDF file or files for convert \n'
                                            'and Select a folder for the converted files!',
                                font='Arial 15', bg='grey', border=2)
        selected_folder.grid(row=1, column=1, sticky='w', padx=15)

        """Buttons"""
        close_button_3 = Button(root4, text='Close', command=close_3)
        close_button_3.grid(column=1, row=2, sticky='n', pady=5, ipady=5, ipadx=5, padx=15)

        root4.mainloop()

    elif len(file_path_pdf_entry.get()) == 0:
        root3 = Tk()
        root3.title('Not files for convert selected')
        root3.geometry('400x80')
        root3.config(bg='grey')
        root3.iconbitmap('icon.ico')

        """Buttons"""
        close_button_2 = Button(root3, text='Close', command=close_2)
        close_button_2.grid(column=1, row=2, sticky='n', pady=5, ipady=5, ipadx=5, padx=30)

        """Labels"""
        selected_files = Label(root3, text='Select PDF file or PDF files for convert', font='Arial 15', bg='grey',
                               border=2)
        selected_files.grid(row=1, column=1, sticky='n', padx=30)

        root3.mainloop()

    elif len(file_path_docx_entry.get()) == 0:
        root2 = Tk()
        root2.title('Not files folder selected')
        root2.geometry('400x80')
        root2.config(bg='grey')
        root2.iconbitmap('icon.ico')

        """Buttons"""

        close_button = Button(root2, text='Close', command=close)
        close_button.grid(column=1, row=2, sticky='n', pady=5, ipady=5, ipadx=5)

        """Labels"""
        selected_folder = Label(root2, text='Select a folder for the converted files!', font='Arial 15', bg='grey',
                                border=2)
        selected_folder.grid(row=1, column=1, sticky='w', padx=30)

        root2.mainloop()

    else:

        root5 = Tk()
        root5.iconbitmap('icon.ico')
        root5.title('Important!')
        root5.config(bg='grey')

        """Label"""
        warning_message_label = Label(root5, text='Make sure, that you don"t have any .docx file \n '
                                                  'in the selected folder with name Docx1, 2, 3, 4..., \n '
                                                  'because, maybe it will be replaced', font='Impact 20', bg='grey')
        warning_message_label.grid(column=1, row=1)

        """Buttons"""

        close_button_4 = Button(root5, text='Continue', command=close_5, border=2, font='Arial 10')
        close_button_4.grid(column=1, row=2, sticky='se', padx=180, ipady=5, ipadx=10)

        cancel_button = Button(root5, text='Cancel', command=close_4, border=2, font='Arial 10')
        cancel_button.grid(column=1, row=2, sticky='sw', padx=180, ipady=5, ipadx=10)

        root5.mainloop()
    ##################


# Close the program window
def close_window():
    sys.exit()


# take you to a webpage, to tell you, how to check your file or files
def callback():
    webbrowser.open_new(r"https://support.policystat.com/hc/en-us/articles"
                        r"/207993346-How-can-I-tell-if-my-PDFs-are-text-based-or-not-")


"""Labels"""

program_use_label = Label(text='PDF to Docx', font='Impact 40', bg='white', fg='#1E90FF')
program_use_label.grid(column=2, row=1, sticky='n', pady=50, padx=120)

check_label = Label(text='*This converter can only convert text based pdf \n cant convert image based pdf',
                    bg='grey')
check_label.grid(row=7, column=2, pady=10)


"""Entries"""

# PDF file entry
file_path_pdf_entry = Entry(border=5)
file_path_pdf_entry.grid(ipadx=90, ipady=4, padx=20, sticky='nw', column=2, pady=1, row=2)

# Docx file entry
file_path_docx_entry = Entry(border=5)
file_path_docx_entry.grid(column=2, ipady=4, ipadx=90, padx=20, sticky='nw', pady=70, row=3)


"""Buttons"""

converter_button = Button(text='Convert', bg='#1E90FF', fg='white', font='impact 20', border=5,
                          command=convert_button_function)
converter_button.grid(padx=175, sticky='s', ipady=5, ipadx=10, column=2, row=4)

select_pdf_file = Button(text='Select PDF files', fg='black', bg='white', border=3,
                         command=pdf_file_location)
select_pdf_file.grid(column=2, sticky='ne', row=2, pady=6, padx=60)

select_new_file_folder = Button(text='Select new files folder', fg='black', bg='white', border=3,
                                command=docx_folder_location)
select_new_file_folder.grid(column=2, sticky='ne', row=3, pady=74, padx=26)

quit_button = Button(text='Quit', bg='#1E90FF', fg='white', font='impact 20', border=5, command=close_window)
quit_button.grid(row=6, column=2, pady=30, ipadx=30)

difference_button = Button(text='how to check my file?', bg='grey', fg='black', command=callback)
difference_button.grid(column=2)

root.mainloop()

```
This time, i tried to converted 13 pdf files at once. No problem at all.

I hope this time, i pulled request in the right place.

dothinking committed:
initial version provided by @JoHnTsIm

dothinking commented:
Great improvement and good progress. We can go a bit deeper, in a GitHub-like way.

GitHub is a platform not only for sharing code but for collaboration, as you did now.  If you're familiar with version control like `git`, you must aware that the main code is hosted in a mater branch, `master` in most cases; and develop branches are created for specific issue or enhancement, like `gui` for this topic. Even though you don't have write access to `dothinking/pdf2docx`, GitHub allows anyone to contribute to the repo. First, you need to fork it to your scope, `JoHnTsIm/pdf2docx`, where you can do anything you want, e.g. apply the changes directly to https://github.com/JoHnTsIm/pdf2docx/blob/gui/pdf2docx/gui.py. After that, your `JoHnTsIm:gui` branch is latest, while `dothinking:master` falls behind. Finally, your pull a request to merge your `JoHnTsIm:gui` to `dothinking:master`, where we can discuss further on the fix/enhancement, like we did just now. After I accept the request, your contribution is included in the master branch of `pdf2docx`.

You can do this with `git`, or GitHub GUI if you're not familiar with it yet.

1. Go to `gui.py` in your `gui` branch https://github.com/JoHnTsIm/pdf2docx/blob/gui/pdf2docx/gui.py

Or you can do it this way: switch to `gui` branch, and navigate to pdf2docx/gui.py

![image](https://user-images.githubusercontent.com/13885442/118429779-06b24d00-b705-11eb-8547-dd302366cd18.png)


2. Edit the file,

![image](https://user-images.githubusercontent.com/13885442/118429854-3e20f980-b705-11eb-9437-64d9534d1eb6.png)

and commit to `gui` branch:

![image](https://user-images.githubusercontent.com/13885442/118429950-83452b80-b705-11eb-9703-11739d219e68.png)

3. Go to your `gui` branch and pull a request

![image](https://user-images.githubusercontent.com/13885442/118435370-70385880-b711-11eb-9329-0ba4531a10c7.png)


Note to push you `gui` branch to `dothinking/master`

![image](https://user-images.githubusercontent.com/13885442/118435885-63683480-b712-11eb-9be4-0e37838457ab.png)


---

I hope this won't make you back. Of course, the collaboration itself is most important, rather than the way how we do it, but I believe this would let you play better with GitHub in the future.

JoHnTsIm committed:
improvment on gui.py

improvments
-------------
1. Now if the user tries to press convert without having select at least one file for convert, its gonna get this pop up window with this message.

![3](https://user-images.githubusercontent.com/84143748/118415240-f9b83e00-b6b1-11eb-97ed-c64a6df665ad.PNG)


2. Now if the user tries to press convert without having seleet a folder to save the converted files, its gonna get this pop up window with this message

![2](https://user-images.githubusercontent.com/84143748/118415309-4734ab00-b6b2-11eb-9f7c-eb088f6ec5a9.PNG)


3. Now if the user tries to press convert without having select a folder to save the converted files and without having select at least one file for convert, its gonna get this window with this message

![1](https://user-images.githubusercontent.com/84143748/118415360-882cbf80-b6b2-11eb-8302-692e79772811.PNG)


4. Now if user do it all and then press convert, its gonna get this pop up window.

![4](https://user-images.githubusercontent.com/84143748/118415400-c88c3d80-b6b2-11eb-85c8-42956c079df3.PNG) 
if the user selects continue the convert its gonna start, otherwise if the user selects cancel the convert its gonna canceled.

JoHnTsIm committed:
Update gui.py

improvments
-------------
1. Now if the user tries to press convert without having select at least one file for convert, its gonna get this pop up window with this message.

![3](https://user-images.githubusercontent.com/84143748/118415240-f9b83e00-b6b1-11eb-97ed-c64a6df665ad.PNG)


2. Now if the user tries to press convert without having seleet a folder to save the converted files, its gonna get this pop up window with this message

![2](https://user-images.githubusercontent.com/84143748/118415309-4734ab00-b6b2-11eb-9f7c-eb088f6ec5a9.PNG)


3. Now if the user tries to press convert without having select a folder to save the converted files and without having select at least one file for convert, its gonna get this window with this message

![1](https://user-images.githubusercontent.com/84143748/118415360-882cbf80-b6b2-11eb-8302-692e79772811.PNG)


4. Now if user do it all and then press convert, its gonna get this pop up window.

![4](https://user-images.githubusercontent.com/84143748/118415400-c88c3d80-b6b2-11eb-85c8-42956c079df3.PNG) 
if the user selects continue the convert its gonna start, otherwise if the user selects cancel the convert its gonna canceled.

JoHnTsIm committed:
Add a folder for the gui

add a folder for the gui that haves, inside the gui.py and the image that needs in order to work.

JoHnTsIm committed:
Add the Gui.py to the folder Gui

JoHnTsIm committed:
Delete gui.py 

delete the gui.py because i put it in a folder with an icon, that need in order for the Gui.py to work, the code is exaxlty the same
